atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1039,8 +1045,10 @@ void WebContents::HandleKeyboardEvent(
   }
 }
 
-void WebContents::EnterFullscreenModeForTab(content::WebContents* source,
-                                            const GURL& origin) {
+void WebContents::EnterFullscreenModeForTab(
+    content::WebContents* source,
+    const GURL& origin,
+    const blink::WebFullscreenOptions& options) {
   auto permission_helper =
       WebContentsPermissionHelper::FromWebContents(source);
   auto callback = base::Bind(&WebContents::OnEnterFullscreenModeForTab,
```

Comment:
this needs to bind to `options` so we can return the same value below in `OnEnterFullscreenModeForTab`

Reply:
Fixed in https://github.com/brave/muon/commit/db6450acd932f510b840224fc6365bbefd504285.